buildtools/cargo/cargo.go

```diff
@@ -0,0 +1,115 @@
+package cargo
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+
+	"github.com/fossas/fossa-cli/errors"
+	"github.com/fossas/fossa-cli/files"
+	"github.com/fossas/fossa-cli/graph"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+type manifest struct {
+	Workspace         workspace
+	Dependencies      map[string]interface{} `toml:"dependencies"`
+	BuildDependencies map[string]interface{} `toml:"build-dependencies"`
+	Package           Package
+}
+
+type workspace struct {
+	Members []string
+}
+
+// lockfile contains the Projects in a dep lockfile and a corresponding map for retrieving project data.
```

Comment:
Is "and a corresponding map for retrieving project data" still supposed to be here? Seems that `type lockfile` used to have another field that you then removed?

Reply:
Yea good catch, that's exactly what happened. Removing this comment entirely now as `lockfile` isn't exported and it only has 1 field.